packages/@dataform/cli/BUILD

```diff
@@ -85,11 +85,23 @@ pkg_bundle(
     ],
 )
 
+add_license_header_to_file(
+    name="bundle_with_license",
+    from_file=":bundle_no_license",
+    to_file=":bundle.js",
```

Comment:
these aren't labels so you don't want : in there, that's probably why tests are failing

Reply:
Tests were also failing because the Node shebang for the CLI was getting in the way; have moved license to below it.